install/operator/pkg/syndesis/configuration/configuration.go

```diff
@@ -302,7 +302,9 @@ func GetProperties(ctx context.Context, file string, clientTools *clienttools.Cl
 	}
 
 	if ac, err := capabilities.ApiCapabilities(apiClient); err != nil {
-		return nil, err
+		if apiClient != nil {
+			return nil, err
+		}
```

Comment:
For my understanding (sure it makes sense), can you explain why this change?

Reply:
Same as previous comment